notebooks/hive/PyHive.ipynb

```diff
@@ -0,0 +1,726 @@
+       " ('6048 OGDEN NASH WAY', 'SACRAMENTO', 95842, 'CA', 3, 2, 1104.0, 'Residential', 'Wed May 21 00:00:00 EDT 2008', 90895.0, 38.68166, -121.35171),\n",
+       " ('2561 19TH AVE', 'SACRAMENTO', 95820, 'CA', 3, 1, 1177.0, 'Residential', 'Wed May 21 00:00:00 EDT 2008', 91002.0, 38.53509, -121.48137),\n",
+       " ('11150 TRINITY RIVER DR Unit 114', 'RANCHO CORDOVA', 95670, 'CA', 2, 2, 941.0, 'Condo', 'Wed May 21 00:00:00 EDT 2008', 94905.0, 38.62119, -121.27055),\n",
+       " ('7325 10TH ST', 'RIO LINDA', 95673, 'CA', 3, 2, 1146.0, 'Residential', 'Wed May 21 00:00:00 EDT 2008', 98937.0, 38.70091, -121.44298)]"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%sql\n",
+    "select * from sacramento_properties_ext limit 10"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load data into a Managed Table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section we are going to load the data in a managed table. The table is going to be partitioned by ZIP and stored in ORC (a columnar storage format)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql\n",
+    "CREATE TABLE sacramento_properties(\n",
+    "street string,\n",
+    "city string,\n",
+    "state string,\n",
+    "beds int,\n",
+    "baths int,\n",
+    "sq__ft float,\n",
+    "sales_type string,\n",
+    "sale_date string,\n",
+    "price float,\n",
+    "latitude float,\n",
+    "longitude float)\n",
+    "PARTITIONED by (zip int)\n",
+    "STORED AS ORC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Done.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<table>\n",
+       "    <tr>\n",
+       "        <th>tab_name</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "        <td>sacramento_properties</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "        <td>sacramento_properties_ext</td>\n",
+       "    </tr>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "[('sacramento_properties',), ('sacramento_properties_ext',)]"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%sql\n",
+    "show tables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default we don't allow dynamic inserts. This means that we should specify, for which row, which partition it belongs to. This to avoid that a mistake in the insert query creates a huge number of directories. \n",
```

Comment:
@SirOibaf `...for each row...`
 `This is meant to avoid having a wrong insert query create a huge...`